src/main_thread/decrypt/__tests__/__global__/get_license.test.ts

```diff
@@ -41,6 +42,37 @@ describe("decrypt - global tests - getLicense", () => {
     jest.restoreAllMocks();
   });
 
+  it("should retry after an increased time at each retry", () => {
+    /*
```

Comment:
Because this test now only tests the `getRetryDelay` util, it may be more at its place outside that `__tests__/__global__` directory which tests behavior for the whole `ContentDecryptor`.

It's probably more a test for `src/utils` to put in `src/utils/__tests__`

Note: I rebased your branch because I had to rebase `dev` on `stable`, you'll have to reset on the remote branch :s...

Reply:
I have made the change